browser/mojSortableTable.ts

```diff
@@ -159,7 +162,25 @@ class PersistentSortOrder {
   private static forceSortOrder(table: HTMLElement, sortOrder: SortOrderDTO) {
     const headers = table.querySelectorAll('thead th') as NodeListOf<HTMLTableCellElement>
 
-    const header = Array.from(headers).find(aheader => aheader.dataset.persistentId === sortOrder.columnPersistentId)
+    let header: HTMLTableCellElement | undefined
+
+    // polyfill for IE11, which doesn't support `.from`
+    if (!Array.from) {
```

Comment:
What the benefit of having the if statement and instead just using the older functionality?

Reply:
I left this one as-is (unlike the older functionality above) as it was easier to store this in a variable and use that conditionally.

I think the older functionality is a bit ugly, and not very readable, so thought I'd leave the newer version there for newer browsers.

I'm not actually sure what the best approach is, here - I wonder if there are performance implications for each implementation?